contracts/custom/DecentDAOVoting.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity =0.8.19;
+
+import { LinearERC20Voting } from "../azorius/LinearERC20Voting.sol";
+import { VotesERC20Wrapper } from "../VotesERC20Wrapper.sol";
+
+interface Escrow {
+    function getBalance(address _address, uint _block) external view returns (uint);
```

Comment:
The Lock & Release contract allows any token to be locked within it, so could this function take a third parameter for the token address?

```solidity
function getBalance(address _token, address _address, uint _block) external view returns (uint);
```

Reply:
I think the lock contract should return a sum of their voting power, regardless of what tokens they have locked.  The strategy won't know what assets are worth voting power, or what their addresses are

Reply:
Are you saying that if my address holds 10 locked DCNT and 1,000,000 locked SHIT then my voting power is equal to 1,000,010 on Decent's Fractal instance?

That doesn't sound right, or maybe I misinterpreted your suggestion.

The strategy must know the DCNT token address, right? It should be summing the unlocked DCNT token balance in a user's address with the that user's locked DCNT balance in this lock contract.

Reply:
ohh, I think I see what you're saying now, I think I misunderstood what you were saying

So escrow will have the `getBalance()` as you suggested, then the strategy will call `getBalance()` with the governance token address

that make sense to me, just pushed up a change to handle that @adamgall

Reply:
Phew, perfect, yes that commit looks like what I was hoping for, thanks!